resources/views/admin/users/delete.twig

```diff
@@ -0,0 +1,14 @@
+{% extends "layouts.admin" %}
+{% block title %}{{ trans('admin::users.delete_title') }}{% endblock %}
+{% block contents %}
+    <div class="admin-body__header">
+        <h1 class="admin-body__header__title">{{ trans('admin::users.delete_title') }}</h1>
+    </div>
+    <div class="admin-body__content">
+        <p>Are you sure you want to delete the user "{{ user.name }}"?</p>
+        {{ form_open({'route': ['admin.users.delete', user.id], 'method': 'post'}) }}
+        <a href="{{ url_route('admin.users') }}" class="button button--secondary"><i class="fa fa-arrow-left"></i> {{ trans('admin::general.back') }}</a>
```

Comment:
You are mixing link with form. Actions like user deletions (database data manipulations) should be sent over POST.  You can use delete button partial `admin/partials/delete_button.twig`